tests/test_config.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017 Facundo Batista
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranties of
+# MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For further info, check  https://github.com/facundobatista/recordium
+
+import os
+from unittest import TestCase, mock
+
+from recordium import config
+
+class TestConfig(TestCase):
+    """Tests for config module."""
+
+    def setUp(self):
+        """Set up. Patch FILEPATH. Creates a fake config"""
+        self.config_patch = mock.patch("recordium.config.FILEPATH")
+        config.FILEPATH = self.config_patch.start()
+        config.FILEPATH = "/tmp/recordium.cfg"
+        self.config_test = config._Config()
+
+    def tearDown(self):
+        """Remove temporary config file. """
+        if os.path.exists(config.FILEPATH):
+            os.remove(config.FILEPATH)
```

Comment:
Please, store the tmp filename in an instance attribute, self.temp_filename, and call os.remove() on that. 

Otherwise, it's error prone: say you remove by accident a patcher line, then you run the tests, and it removed your REAL config!!